README.md

```diff
@@ -37,7 +37,7 @@ Steps (assumes a linux system)
 
 3. Go into the approval\_frame directory. `cd approval_frame`.
 
-4. For the registration procedure to work correctly, fill-in the following email configuration in approval\_frame/settings.py
+4. For the registration procedure to work correctly, create a new file approval\_frame/local\_settings.py and add in the following email configuration.
```

Comment:
To be more specific, could we say

For the registration procedure to work correctly *in development (as we don't want your email ID and password to be committed to github)*, create a new file approval\_frame/local\_settings.py and add in the following email configuration.